homeassistant/components/raincloud/__init__.py

```diff
@@ -62,7 +63,7 @@
     "next_cycle": "",
     "rain_delay": "days",
```

Comment:
```suggestion
    "rain_delay": TIME_DAYS,
```
(+ the import)

Reply:
Changed in https://github.com/home-assistant/home-assistant/pull/32065/commits/59c869eaff07bee606b3878266b699fcb0578767